jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -1,40 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?><?pde version="3.6"?>
-<target includeMode="feature" name="jbosstoolstarget-4.21.3.Final-SNAPSHOT">
+<target includeMode="feature" name="jbosstoolstarget-4.23.0.AM1-SNAPSHOT">
   <locations>
     <!-- don't forget to increment these files when moving up a version: build.xml, *.target -->
     
     
     <location includeAllPlatforms="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
-      <repository location="https://download.jboss.org/jbosstools/updates/requirements/orbit/R20210825222808/"/>
+      <repository location="https://download.jboss.org/jbosstools/updates/requirements/orbit/S20220215213605/"/>
 
       <!-- Global Orbit bundles required by upstream and/or downstream -->
-      <unit id="com.google.gson" version="2.8.7.v20210624-1215"/>
+      <unit id="com.google.gson" version="2.8.9.v20220111-1409"/>
       <unit id="com.google.guava" version="27.1.0.v20190517-1946"/>
       <unit id="com.google.guava" version="30.1.0.v20210127-2300"/>
       <unit id="jakarta.activation" version="2.0.0.v20201118-1818"/>
       <unit id="jakarta.servlet" version="5.0.0.v20210105-0527"/>
       <unit id="jakarta.websocket" version="2.0.0.v20210105-1550"/>
-      <unit id="javax.activation" version="1.2.2.v20201119-1642"/>
+      <unit id="javax.activation" version="1.1.0.v201211130549"/>
```

Comment:
Is that expected ? Seems javax.activation is duplicated in several places, maybe cleanup ?

Reply:
multiple versions are expected and i review them regularly. reverted to use the correct version needed by upstream/components.